src/util.js

```diff
@@ -107,3 +107,80 @@ export function format_memory_and_limit(usage, limit) {
         return _("");
     }
 }
+
+export function updateContainers() {
+    let newContainers = {};
+    let newContainersStats = {};
+    let newContainersMeta = [];
+    let inspectRet = [];
+    let containerStatsRet = [];
+    let countRun;
+    return new Promise((resolve, reject) => {
+        varlinkCall(PODMAN, "io.podman.ListContainers")
+                .then(reply => {
+                    newContainersMeta = reply.containers;
+                    if (!newContainersMeta) {
+                        console.log(newContainersMeta);
+                        resolve({newContainers: newContainers, newContainersStats: newContainersStats});
+                    }
+                    inspectRet = newContainersMeta.map((container) => {
+                        return new Promise((resolve, reject) => {
+                            varlinkCall(PODMAN, "io.podman.InspectContainer", {name: container.id})
+                                    .then(reply => {
+                                        resolve(JSON.parse(reply.container));
+                                    })
+                                    .catch(ex => reject(new Error("Failed to do InspectContainer call:", ex, JSON.stringify(ex))));
+                        });
+                    });
+                })
+                .then(function() {
+                    const containerCount = newContainersMeta.length;
+                    Promise.all(inspectRet)
+                            .then((inspectRet) => {
+                                let runEle = newContainersMeta.filter((ele) => {
+                                    return ele.status === "running";
+                                });
```

Comment:
Drop the `{ return [...]; }` for single-expression handlers, please: `filter(ele => ele.status === "running");`